### PR TITLE
 added new arrival message type 141

### PR DESCRIPTION
**Pull request by sonisn1988**

### CODE REVIEW

File: test/uk/gov/hmrc/transitmovementsauditing/models/AuditTypeSpec.scala [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ */
+
+package uk.gov.hmrc.transitmovementsauditing.models
+
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.must.Matchers
+
+class AuditTypeSpec extends AnyFreeSpec with Matchers {
+
+  "every AuditType must match the object name" - AuditType.values.foreach {
```
Comment (liandra-giacomine) [inline test/uk/gov/hmrc/transitmovementsauditing/models/AuditTypeSpec.scala:24]: Looks good to me, just one question:
Do we use .toString anywhere? Just wondering if these tests are useful?

Comment (sonisn1988) [reply test/uk/gov/hmrc/transitmovementsauditing/models/AuditTypeSpec.scala:24]: yes with API test cases Dan has written same way.
https://github.com/hmrc/common-transit-convention-traders/blob/main/test/v2/models/AuditTypeSpec.scala

Comment (liandra-giacomine) [reply test/uk/gov/hmrc/transitmovementsauditing/models/AuditTypeSpec.scala:24]: @drnaylor-oc how come we verify that toString matches the name? I'm sure there's a good reason but I just want to understand it :)

Comment (drnaylor) [reply test/uk/gov/hmrc/transitmovementsauditing/models/AuditTypeSpec.scala:24]: the comment explains it:

`// toString prints the case object's identifier, so this is what we want`

If you have a `case object`, the name of the case object is also the output of `toString`. I was verifying that the audit type name matched the name of case object, there was a case where it wasn't which is why the test was introduced. Of course, the alternative would have just been to use `toString` where we want to get the audit type out, but we'd gone down this road before I realised we could do that.

Basically, if we have

```scala
case object AnObject
```

then `AnObject.toString` is `"AnObject"`.

Comment (liandra-giacomine) [reply test/uk/gov/hmrc/transitmovementsauditing/models/AuditTypeSpec.scala:24]: @drnaylor-oc Yeah I could see what the test was testing, but I was trying to understand where toString was being used. I can see it's being used in auditTypePathBindable, so all makes sense now. Thanks :)

**Review by liandra-giacomine — APPROVED** [review]
